demo.html

```diff
@@ -97,7 +97,7 @@ <h1 class="font-weight-light mt-5">Eclipse PASS Demo</h1>
             <p class="mt-5 mb-0">
               You can find instructions on how to use the demo instance <a href="https://github.com/eclipse-pass/main/blob/main/docs/user/README.md">here</a>. Interested in running pass at your institution? <a href="contact.html">Contact us!</a>
```

Comment:
Not part of this change, but should `pass` here instead be `PASS`?

Reply:
Possibly yes.